Frameworks/QuartzCore/CALayer.mm

```diff
@@ -569,6 +569,7 @@ - (void)display {
         }
 
         if (!target) {
+#if 0 // TODO(DH): GH#1125 evaluate the interplay between context and displaytexture.
             if ((priv->isOpaque && priv->_backgroundColor == nil) || (priv->backgroundColor.a == 1.0 && 0)) {
```

Comment:
Removed!

---
In reply to: [83078503](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r83078503) [](ancestors = 83078503,83068253,82922896)